Makefile

```diff
@@ -1,4 +1,5 @@
 langNames.db: language_names/scraped-cldr.tsv language_names/scraped-sil.tsv language_names/manual-fixes.tsv language_names/manual-additions.tsv language_names/variants.tsv
+	python3 language_names/sort.py $^
```

Comment:
This should not be a separate script. This can be done absolutely trivially with Bash. Moreover, the `scraper-*.py` scripts should provide sorted output. They should also use `DictWriter` instead of manually creating a TSV.

Reply:
Okay I will remove this script